examples/get_subaccounts.py

```diff
@@ -4,27 +4,20 @@
 This script is equivalent to the test_fetch_first_subaccount() test in tests/test_main.py.
 """
 
-from pathlib import Path
-
-import click
-
 from derive_client.enums import Environment
 from derive_client.http_client import HttpClient as DeriveClient
+from tests.conftest import TEST_PRIVATE_KEY, TEST_WALLET
 
 
-@click.command()
-@click.option('--signer-key-path', required=True, help='Path to signer key file')
-@click.option('--derive-sc-wallet', required=True, help='Derive SC wallet address')
-def main(signer_key_path, derive_sc_wallet):
-    key_file = Path(signer_key_path)
-    if not key_file.exists():
-        click.echo(f"Signer key file not found: {signer_key_path}")
```

Comment:
can you please revert this, its quite a useful tool prior haha

Reply:
That makes no sense to me. This is an example, not a cli tool. `if __name__ == "__main__"` didn't run since no path and address were provided. You want to run `main()` with a path to a test key and provide a test wallet address? What is the purpose of that, as, again, it's not part of a cli tool?